store/sql_user_store.go

```diff
@@ -24,6 +24,7 @@ func NewSqlUserStore(sqlStore *SqlStore) UserStore {
 		table.ColMap("Username").SetMaxSize(64)
 		table.ColMap("Password").SetMaxSize(128)
 		table.ColMap("AuthData").SetMaxSize(128)
```

Comment:
For Facebook we can use direct user_ids which are not very long and all the Google examples I could find have the token about ~20 in length